### PR TITLE
Little Fixes

### PR DESCRIPTION
**Pull request by gchoqueux**

### CODE REVIEW

File: src/Core/Layer/Layer.js [hunk 1]
```diff
@@ -115,12 +115,17 @@ Layer.prototype.constructor = Layer;
...
-            if (imagery.sequence === newIndex) {
-                imagery.sequence = oldIndex;
-                layer.sequence = newIndex;
-                break;
+            if (imagery.id === layer.id) {
+                imagery.sequence = newIndex;
+            } else if (imagery.sequence > oldIndex && imagery.sequence <= newIndex) {
+                imagery.sequence--;
+            } else if (imagery.sequence >= newIndex && imagery.sequence < oldIndex) {
+                imagery.sequence++;
```
Comment (peppsac) [inline src/Core/Layer/Layer.js]: You're changing the semantic of the `moveLayerToIndex` - I'm not against it but maybe you could add a comment stating the expected behavior of this function?

File: src/Core/Prefab/GlobeView.js [hunk 2]
```diff
@@ -258,7 +258,7 @@ GlobeView.prototype.constructor = GlobeView;
 GlobeView.prototype.addLayer = function addLayer(layer) {
     if (layer.type == 'color') {
         const colorLayerCount = this.getLayers(l => l.type === 'color').length;
-        layer.sequence = colorLayerCount - 1;
+        layer.sequence = colorLayerCount;
```
Comment (peppsac) [inline src/Core/Prefab/GlobeView.js:261]: I was going to submit a PR to fix this one :)

**Review by peppsac — APPROVED** [review]
Looks good.
Add the suggested comment and we're good to go.

**Comment by gchoqueux** [comment]
@peppsac Done

**Comment by peppsac** [comment]
Thanks :)

**Pull request merged**